src/sentry/api/endpoints/project_metrics.py

```diff
@@ -52,6 +52,14 @@ class ProjectMetricsVisibilityEndpoint(ProjectEndpoint):
     publish_status = {"PUT": ApiPublishStatus.EXPERIMENTAL}
     owner = ApiOwner.TELEMETRY_EXPERIENCE
 
+    def _get_sanitized_tags(self, request: Request) -> Sequence[str]:
+        tags = request.data.get("tags")
+        if not tags:
+            raise InvalidParams("You must supply at least one tag to block")
+
+        # For now, we want to disallow any glob in the tags, since it might cause issues in Relay.
+        return [tag.replace("*", "") for tag in tags]
```

Comment:
This is definitely not a good UX, because we are silently changing the actual value of the tag which we are blocking and from the UI the user won't be able to notice that unless, they check the API responses.